Themes/default/PersonalMessage.template.php

```diff
@@ -463,11 +464,11 @@ function template_single_pm($message)
 				</ul>
 			</div><!-- .poster -->
 			<div class="postarea">
-				<div class="flow_hidden">
-					<div class="keyinfo">
-						<h5 id="subject_', $message['id'], '">
+				<div class="keyinfo">
+					<div class="postinfo">
```

Comment:
I think in this case we don't want to use `postinfo`, because we really do want the lines inside this div to remain on separate lines.

Reply:
This helps using a single css selector since it’d be the same as the Display template

Reply:
Ok I finally got some sleep, and so show be fixed. 
The subject was supposed to go in before the postinfo

Reply:
See new review.